src/System.Drawing.Common/src/System/Drawing/Font.Unix.cs

```diff
@@ -215,7 +215,7 @@ public static Font FromHfont(IntPtr hfont)
         public IntPtr ToHfont()
         {
             if (_nativeFont == IntPtr.Zero)
-                throw new ArgumentException("Object has been disposed.");
+                throw new ArgumentException(SR.ObjectDisposed);
```

Comment:
Not introduced by your PR, but would this be better as `ObjectDisposedException`? That would bring it inline with Icon's ToBitmap (below).

Reply:
The purpose was not to change the text of messages, just move that messages to resource file